src/main.c

```diff
@@ -419,7 +669,33 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
 
             AddMetadata = FALSE;
             memset(&PacketMetadata, 0, sizeof(DOT11_EXTSTA_RECV_CONTEXT));
-        } else {
+        } 
+        else if (VMSwitchPacketFragment != NULL) {
```

Comment:
Follow the existing brace placement conventions in the code.

if (foo) {
    bar();
} else if (baz) {
    bip();
}

Reply:
fixed